qiskit/algorithms/evolvers/classical_methods/scipy_imaginary_evolver.py

```diff
@@ -0,0 +1,192 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Interface for Classical Quantum Real Time Evolution."""
+from typing import Tuple, Optional, Union
+from scipy.sparse.linalg import expm_multiply
+import numpy as np
+import scipy.sparse as sp
+from qiskit.algorithms.list_or_dict import ListOrDict, List
+from qiskit import QuantumCircuit
+from qiskit.opflow import StateFn
+from qiskit.quantum_info.states import Statevector
+
+
+from ..evolution_problem import EvolutionProblem
+from ..evolution_result import EvolutionResult
+from ..imaginary_evolver import ImaginaryEvolver
+from ...list_or_dict import ListOrDict
+
+
+class ScipyImaginaryEvolver(ImaginaryEvolver):
+    """Classical Evolver for imaginary time evolution."""
+
+    def __init__(
+        self,
+        timesteps: int = 1,
+    ):
+        """
+        Args:
+            timesteps: The number of timesteps to get data from the observables. At the end of each
+                       step the state will be renormalized. Note that increassing the ammount of
+                       timesteps will not increase the accuracy of the integration but can avoid
+                       overflow errors for hamiltonians with high eigenvalues.
```

Comment:
I think it is important for the user to know that increasing timesteps will not result in more accurate results, but that at the same time they need to set a number high enough in order to avoid overflows. Otherwise they might either set timesteps as too high expecting to get better results, or as too low, getting NaNs and not knowing where it comes from.